pkg/agent/config.go

```diff
@@ -16,10 +16,16 @@ type Config struct {
 	Metrics   MetricsConfig    `json:"metrics"`
 	Scheduler SchedulerConfig  `json:"scheduler"`
 	Monitor   MonitorConfig    `json:"monitor"`
+	NeonVM    NeonVMConfig     `json:"neonvm"`
 	Billing   *billing.Config  `json:"billing,omitempty"`
 	DumpState *DumpStateConfig `json:"dumpState"`
 }
 
+type RateThresholdConfig struct {
```

Comment:
Validation of the added configs should be added to `(*config).validate()` below

Reply:
fixed in 26f1567d463e630c4da6f9d10d29b843101901db